scripts/migrate-api-pages.mjs

```
@@ -0,0 +1,344 @@
+ * Delete this script once the last category is migrated; the permanent
+ * `docs-section-order` conformance check holds the line afterwards.
+ */
+import { readFile, writeFile, readdir } from 'node:fs/promises';
+import { join, relative, dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { createRequire } from 'node:module';
+
+import {
+  sectionSpans,
+  sectionBody,
+  splitLines,
```

ESLint

[error] 'splitLines' is defined but never used.

(no-unused-vars)